www/apps/resources/app/recipes/subscriptions/examples/standard/page.mdx

```diff
@@ -0,0 +1,2288 @@
+      resolve: "./modules/subscription",
+    },
+  },
+})
+```
+
+### Further Read
+
+- [How to Create a Module](!docs!/basics/modules-and-services)
+- [How to Create a Data Model](!docs!/basics/data-models)
+- [Learn more about the service factory](!docs!/advanced-development/modules/service-factory).
+
+---
+
+## Step 2: Define Links
+
+In this step, you’ll define links between the Subscription Module’s `Subscription` data model and the data models of Medusa’s commerce modules.
+
+### Define a Link to Cart
+
+To link a subscription to the cart used to make the purchase, create the file `src/links/subscription-cart.ts` with the following content:
+
+```ts title="src/links/subscription-cart.ts"
+import { defineLink } from "@medusajs/utils"
+import SubscriptionModule from "../modules/subscription"
+import CartModule from "@medusajs/cart"
+
+export default defineLink(
+  SubscriptionModule.linkable.subscription,
+  CartModule.linkable.cart
+)
+```
+
+This defines a link between the `Subscription` data model and the Cart Module’s `Cart` data model.
+
+<Note title="Tip">
+
+When you create a new order for the subscription, you’ll retrieve the linked cart and use the same shipping and payment details the customer supplied when the purchase was made.
+
+</Note>
+
+### Define a Link to Customer
+
+To link a subscription to the customer who purchased it, create the file `src/links/subscription-customer.ts` with the following content:
+
+```ts title="src/links/subscription-customer.ts"
+import { defineLink } from "@medusajs/utils"
+import SubscriptionModule from "../modules/subscription"
+import CustomerModule from "@medusajs/customer"
+
+export default defineLink(
+  {
+    linkable: SubscriptionModule.linkable.subscription,
+    isList: true,
+  },
+  CustomerModule.linkable.customer
+)
+```
+
+This defines a list link to the `Subscription` data model, since a customer may have multiple subscriptions.
+
+### Define a Link to Order
+
+To link a subscription to the orders created for it, create the file `src/links/subscription-order.ts` with the following content:
+
+```ts title="src/links/subscription-order.ts"
+import { defineLink } from "@medusajs/utils"
+import SubscriptionModule from "../modules/subscription"
+import OrderModule from "@medusajs/order"
+
+export default defineLink(
+  SubscriptionModule.linkable.subscription,
+  {
+    linkable: OrderModule.linkable.order,
+    isList: true,
+  }
+)
+```
+
+This defines a list link to the `Order` data model since a subscription has multiple orders.
+
+### Further Reads
+
+- [How to Define a Link](!docs!/advanced-development/modules/module-links)
+
+---
+
+## Step 3: Run Migrations
+
+To create a table for the `Subscription` data model in the database, start by generating the migrations for the Subscription Module with the following command:
+
+```bash
+npx medusa migrations generate subscriptionModuleService
+```
+
+This generates a migration in the `src/modules/marketeplace/migrations` directory.
```

Comment:
This generates a migration in the `src/modules/subscription/migrations` directory.

(`marketeplace` is incorrect)